packages/search-metadata-previews/src/snippet-editor/ModeSwitcher.js

```diff
@@ -83,8 +75,6 @@ class ModeSwitcher extends Component {
 		const desktopInputId = desktopModeInputId.length > 0 ? desktopModeInputId : "yoast-google-preview-mode-desktop";
 
 		return ( <Switcher>
-			<SearchPreviewDescription>{ __( "Determine how your post should look in the search results.",
-				"wordpress-seo" ) }</SearchPreviewDescription>
```

Comment:
I move the rendering of the Search description to `SnippetEditor` component, instead of inside the Switcher component